Smalltalk.gitignore

```diff
@@ -1,15 +1,24 @@
 # changes file
 *.changes
+*.chg
```

Comment:
Represents the same as in Pharo and Squeak (*.changes).
When you save something (ctrl+s) this change is add to the chg file, everything you modify is added. This file is complemented by SML and IMG/IMG7 to create the complete image, but what is important to version are the code packages (*.pac or *.pax - *.cls)